eng/common/build.ps1

```diff
@@ -32,7 +32,7 @@ Param(
 # Unset 'Platform' environment variable to avoid unwanted collision in InstallDotNetCore.targets file
 # some computer has this env var defined (e.g. Some HP)
 if($env:Platform) {
-  $env:Platform=""  
```

Comment:
This file is not really part of our repo; it's copied here from https://github.com/dotnet/arcade/blob/main/eng/common/build.ps1. Please revert the change (it will just get overwritten on the next Arcade update).